docs/UserGuide.md

```diff
@@ -93,11 +93,16 @@ The `Code Panel` allows users to see the code attributed to a specific author. C
 * Non-trivial code segments that are not written by the selected author are hidden by default, but you can toggle them by clicking on the `...` icon.
 
 ### Commits Panel
+![Commits panel](/docs/images/commits-panel.png)
```

Comment:
this needs to be resize smaller so that the words doesn't seem excessively bigger than the font of documentation.